### PR TITLE
Updates outdated issue template

### PR DESCRIPTION
**Pull request by moonso**
## Description

This PR updates the issue template

## Review
- [x] code approved by HS, PG, MR
- [x] "Merge and deploy" approved by MM
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by moonso** [comment]
Ping @barrystokman @patrikgrenfeldt @henrikstranneheim @Mropat I would appreciate if you all have a quick look at this

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -1,30 +1,21 @@
...
 
-Questions: 
-Q: Does the same problem occur for tea?
-A: Yes.
+## This can be closed when
 
-What needs to be done on the planning meeting:
-- Estimate storypoints
-- **Remember to add label *PBL Teamname***
+*Describe what needs to be done for this issue to be closed*
```
Comment (patrikgrenfeldt) [inline .github/ISSUE_TEMPLATE/user-story.md:17]:
```suggestion
- *List the things that needs to be done for this issue to be resolved*
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Review by Mropat — APPROVED** [review]
:fire: :fire:

**Pull request merged**